### DISCUSSION TITLE
add an action to the button to show/hide container.

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2224] [Ideas]
Hello,

I would like to show a container, when a user click on a button :
![image](https://user-images.githubusercontent.com/3023526/127892215-67feba03-f090-4323-93bb-d8abc11a60f6.png)

I try to find a way to do it, so I am using the checkbox, but I think the binding is not in real time?

**Comment by kevmodrome** [comment]
Hi! 

You can do this by wrapping things in a form. Inside the form you add a toggle. Type in an arbitrary field name. Condition the container to show/hide depending on said toggle. A good idea is to add a Form at the top level of your screen that is just for holding different screen states.

Here's an image showing the way to do it:
<img width="1792" alt="toggle" src="https://user-images.githubusercontent.com/534488/127901067-e1c72041-2cdc-433b-9493-e3c8cb88ee94.png">

**Comment by KernOil** [comment]
@oswaldB  to do this with a button I just made two pages one with the charts and one without I have a button on the bottom that says view on one page which will navigate to the page with the charts and a button with hiding one the charts page that will navigate to the page with no charts. it's pretty seamless. you cant tell you loaded a different page.
